src/styles/containers/StartScreen/_export-error-list.scss

```diff
@@ -10,6 +10,7 @@
       margin: 0.75rem;
 
       .icon {
+        flex: 0 0 auto;
```

Comment:
Fixes a small visual glitch where amount error text caused the icon to be smaller or larger